pkg/imager/iso/uefi.go

```diff
@@ -65,7 +64,7 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 
 	printf("preparing raw image")
 
-	efiBootImg := filepath.Join(options.ScratchDir, "efiboot.img")
```

Comment:
this looks wrong, `os.TempDir()` is basically `/tmp`

Reply:
yeh, I just didn't want this file to go in iso

Reply:
i'll use `os.MkdirTemp` to use a temp dir and cleanup afterwards,

Reply:
we could probably just create a directory structure under scratch dir

```
scratch_dir
  efiroot
     EFI
     loader
  efiboot.img
```

I think that would be more clean?

Reply:
:+1: makes more sense